exercises/practice/grade-school/runtests.jl

```diff
@@ -0,0 +1,240 @@
+using Test
+
+include("grade-school.jl")
+
+Student = NamedTuple{(:name, :grade), Tuple{String, Int}}
```

Comment:
I wonder if this should maybe be part of the stub file, to help with discoverability for students.

Reply:
I initially considered the suggesting that, but then I thought that putting it in the stub it might be confusing to students since it's not actually needed in their functions (I usually assume that any boilerplate needs to be used somehow).

In the end, since it's a `NamedTuple`, it behaves just like a normal `Tuple`, but with the added option of accessing the fields directly, so I don't think it needs to be discovered and could be left as a kind of hidden gem.

Reply:
Okay!